openmdao/docs/usr-guide/examples/example_fd.rst

```diff
@@ -264,3 +264,128 @@ Nothing else changes in the original model. When we run it, we get:
 
 So here, `jacobian` is never called in any component as the finite difference
 just executes the components in sequence. This is also as expected.
+
+
+Parallel Finite Difference
+==========================
+
+Suppose you need to calculate a bunch of finite differences, either because
+you have a bunch of different design variables, or maybe just a single design
+variable that happens to be an array.  OpenMDAO has a special `Group` called
+a `ParallelFDGroup` that will allow you to calculate multiple finite differences
+in parallel.
+
+Let's start off our example by creating a `Component` that has array inputs
+and outputs.
+
+.. testcode:: fd_par_example
+
+    import numpy
+    import time
+    from openmdao.api import Problem, Component, ParallelFDGroup, IndepVarComp
+    from openmdao.core.mpi_wrap import MPI
+
+    class ArrayFDComp(Component):
+        """ A simple component takes an array input, produces
+        an array output, and does not provide derivatives.
+
+        Args
+        ----
+        size : int
+            The size of the input and output variables.
+
+        delay : float
+            The number of seconds to sleep during the solve_nonlinear
+            call.
+        """
+
+        def __init__(self, size, delay):
+            super(ArrayFDComp, self).__init__()
+
+            self.delay = delay
+
+            # Params
+            self.add_param('x', numpy.zeros(size))
+
+            # Unknowns
+            self.add_output('y', numpy.zeros(size))
+
+        def solve_nonlinear(self, params, unknowns, resids):
+            """ Doesn't do much.  Just multiply by 3"""
+            time.sleep(self.delay)
+            unknowns['y'] = 3.0*params['x']
+
+The following check is only here so that our doc tests, which don't run
+under MPI, will pass.  In real life, you would never use a `ParallelFDGroup`
+unless you were running under MPI.
+
```

Comment:
You can use the `:hide:` directive to hide display of a code block. Then you don't have to put the paragraph explaining how it is needed to run doctests.